### ISSUE TITLE
Question about group

### ISSUE THREAD

**Issue by Pierre9344** [quarto-ext/lightbox#7]
Hello,

Is it possible to indicate groups for graph generated by one or multiple r code chunk? The idea would be reproduce the behavior of the 3 lines bellow with an argument similar to *'#| lightbox-group: "picture 1"'*

```
![A Lovely Image](mv-1.jpg){group="my-gallery"}
![Another Lovely Image](mv-2.jpg){group="my-gallery"}
![The Last Lovely Image](mv-3.jpg){group="my-gallery"}
```

**Comment by s-lt** [comment]
Thank for the pointer, the following works perfectly for me:
#| out.extra: 'group="my-gallery"'

**Comment by cderv** [comment]
This was solved by #13 and this is now possible. 

Example with several plot in a single chunk 
````markdown
---
title: Simple Lightbox Example
filters:
   - lightbox
lightbox: auto
---

```{r}
#| fig-cap: 
#|   - Caption for first plot
#|   - Caption for second plot
#| lightbox: 
#|   group: cars
#|   description: 
#|     - This is the decription for first graph
#|     - This is the decription for second graph
plot(mtcars)
plot(cars)
```
````
